app/controllers/ontologies_controller.rb

```diff
@@ -159,6 +159,7 @@ def new
 
   def notes
     @notes = @ontology.explore.notes
+    @notes = @notes.uniq { |note| [note.body, note.subject] }
```

Comment:
this is not a solution, 
the issue is why the API return duplicate notes? do they have the same id ? if no, so they are not the same.

Reply:
if we see the API, we see that the IDs are different
![image](https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/29259906/2440c6bf-11b2-44c5-8ee3-53e507a34b0e)

The issue was resolved here https://github.com/ontoportal-lirmm/bioportal_web_ui/issues/167, a time ago. The bug was when we created a note, the event was triggered 4 times, creating multiple notes.

In summary to try to create note, and see it works well and create only one. And remove the current duplicate

Reply:
Yes u r right, the issue was in the creation, and now it's solved